.github/workflows/orca.yml

```diff
@@ -0,0 +1,128 @@
+---
+name: ORCA CI
+on:
+    push:
+        branches: [ main, develop, wip, support/**, ORCA-189-2 ]
+        paths-ignore:
+            - .idea/**
+            - docs/**
+    pull_request:
+        branches: [ develop ]
+        paths-ignore:
+            - .idea/**
+            - docs/**
+    schedule:
+        # Daily at 00:00:00 UTC.
+        # @see https://crontab.cronhub.io/
+        -   cron: "0 0 * * *"
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    continue-on-error: ${{ matrix.allow-failure }}
+    env:
+      ORCA_SUT_NAME: drupal/example
+      ORCA_SUT_BRANCH: main
+      ORCA_PACKAGES_CONFIG: example/tests/packages.yml
+      ORCA_PACKAGES_CONFIG_ALTER: example/tests/packages_alter.yml
+      ORCA_ENABLE_NIGHTWATCH: FALSE
+      # Hardcode path since GITHUB_WORKSPACE can't be used here.
+      # @see https://github.community/t/how-to-use-env-context/16975/9
+      ORCA_SUT_DIR: /home/runner/work/orca/example
+      ORCA_SELF_TEST_COVERAGE_CLOVER: $HOME/build/logs/clover-self.xml
+      ORCA_JOB: ${{ matrix.orca-job }}
+      ORCA_LIVE_TEST: ${{ matrix.orca-live-test }}
+
+    strategy:
+      matrix:
+        orca-job:
+          - STATIC_CODE_ANALYSIS
+          - INTEGRATED_TEST_ON_OLDEST_SUPPORTED
+          - INTEGRATED_TEST_ON_LATEST_LTS
+          - INTEGRATED_TEST_ON_PREVIOUS_MINOR
+          - INTEGRATED_UPGRADE_TEST_FROM_PREVIOUS_MINOR
+          - ISOLATED_TEST_ON_CURRENT
+          - INTEGRATED_TEST_ON_CURRENT
+          - INTEGRATED_UPGRADE_TEST_TO_NEXT_MINOR
+          - ISOLATED_TEST_ON_CURRENT_DEV
+          - INTEGRATED_TEST_ON_CURRENT_DEV
+          - STRICT_DEPRECATED_CODE_SCAN
+          - ISOLATED_TEST_ON_NEXT_MINOR
+          - INTEGRATED_TEST_ON_NEXT_MINOR
+          - ISOLATED_UPGRADE_TEST_TO_NEXT_MAJOR_BETA_OR_LATER
+        php-version: [ "7.3" ]
+        allow-failure: [ false ]
+        include:
+          - orca-job: ISOLATED_TEST_ON_CURRENT
+            php-version: "8.0"
+            allow-failure: false
+
+          - orca-job: INTEGRATED_UPGRADE_TEST_TO_NEXT_MINOR_DEV
+            php-version: "7.3"
+            allow-failure: true
+
+          - orca-job: LOOSE_DEPRECATED_CODE_SCAN
+            php-version: "7.3"
+            allow-failure: true
+
+          - orca-job: DEPRECATED_CODE_SCAN_W_CONTRIB
+            php-version: "7.3"
+            allow-failure: true
+
+          - orca-job: ISOLATED_TEST_ON_NEXT_MINOR_DEV
+            php-version: "7.3"
+            allow-failure: true
+
+          - orca-job: INTEGRATED_TEST_ON_NEXT_MINOR_DEV
+            php-version: "7.3"
+            allow-failure: true
+
+          - orca-job: ISOLATED_UPGRADE_TEST_TO_NEXT_MAJOR_DEV
+            php-version: "7.3"
+            allow-failure: true
```

Comment:
Per https://github.com/acquia/orca/blob/v3.9.0/.travis.yml#L68-L71, our `.travis.yml` currently allows three jobs to fail, but this Actions config adds three more:

```diff
 DEPRECATED_CODE_SCAN_W_CONTRIB
 INTEGRATED_TEST_ON_NEXT_MINOR_DEV
+INTEGRATED_UPGRADE_TEST_TO_NEXT_MINOR_DEV
 ISOLATED_TEST_ON_NEXT_MINOR_DEV
+ISOLATED_UPGRADE_TEST_TO_NEXT_MAJOR_DEV
+LOOSE_DEPRECATED_CODE_SCAN
```

Reply:
Oh I was confused because ORCA's .travis.yml allows _fewer_ failed jobs than the template .travis.yml. That's fine then.